scripts/ci-ar.sh

```diff
@@ -36,13 +36,15 @@ else
     yarn --silent
 
     cd apps-rendering
-
+ 	(
```

Comment:
wonky formatting of braces?

Reply:
I removed the whitespaces before the opening parenthesis.